src/vuefire.js

```diff
@@ -61,11 +85,7 @@ function bind (vm, key, source) {
   if (!isObject(source)) {
     throw new Error('VueFire: invalid Firebase binding source.')
   }
-  // get the original ref for possible queries
-  var ref = source
-  if (typeof source.ref === 'function') {
```

Comment:
Both of us ([Reference](https://firebase.google.com/docs/reference/js/firebase.database.Reference#ref) and [Query](https://firebase.google.com/docs/reference/js/firebase.database.Query#ref)) have a `ref` property, so it needs to check if the `ref` is `object`?